app/resources/country-continent.json

```diff
@@ -0,0 +1 @@
inent_Code": "EU", "Continent_Name": "Europe", "Country_Name": "Isle of Man", "Country_Number": 833, "Three_Letter_Country_Code": "IMN", "Two_Letter_Country_Code": "IM"},{"Continent_Code": "AF", "Continent_Name": "Africa", "Country_Name": "Tanzania, United Republic of", "Country_Number": 834, "Three_Letter_Country_Code": "TZA", "Two_Letter_Country_Code": "TZ"},{"Continent_Code": "NA", "Continent_Name": "North America", "Country_Name": "United States of America", "Country_Number": 840, "Three_Letter_Country_Code": "USA", "Two_Letter_Country_Code": "US"},{"Continent_Code": "NA", "Continent_Name": "North America", "Country_Name": "United States Virgin Islands", "Country_Number": 850, "Three_Letter_Country_Code": "VIR", "Two_Letter_Country_Code": "VI"},{"Continent_Code": "AF", "Continent_Name": "Africa", "Country_Name": "Burkina Faso", "Country_Number": 854, "Three_Letter_Country_Code": "BFA", "Two_Letter_Country_Code": "BF"},{"Continent_Code": "SA", "Continent_Name": "South America", "Country_Name": "Uruguay, Eastern Republic of", "Country_Number": 858, "Three_Letter_Country_Code": "URY", "Two_Letter_Country_Code": "UY"},{"Continent_Code": "AS", "Continent_Name": "Asia", "Country_Name": "Uzbekistan, Republic of", "Country_Number": 860, "Three_Letter_Country_Code": "UZB", "Two_Letter_Country_Code": "UZ"},{"Continent_Code": "SA", "Continent_Name": "South America", "Country_Name": "Venezuela, Bolivarian Republic of", "Country_Number": 862, "Three_Letter_Country_Code": "VEN", "Two_Letter_Country_Code": "VE"},{"Continent_Code": "OC", "Continent_Name": "Oceania", "Country_Name": "Wallis and Futuna", "Country_Number": 876, "Three_Letter_Country_Code": "WLF", "Two_Letter_Country_Code": "WF"},{"Continent_Code": "OC", "Continent_Name": "Oceania", "Country_Name": "Samoa, Independent State of", "Country_Number": 882, "Three_Letter_Country_Code": "WSM", "Two_Letter_Country_Code": "WS"},{"Continent_Code": "AS", "Continent_Name": "Asia", "Country_Name": "Yemen", "Country_Number": 887, "Three_Letter_Country_Code": "YEM", "Two_Letter_Country_Code": "YE"},{"Continent_Code": "AF", "Continent_Name": "Africa", "Country_Name": "Zambia, Republic of", "Country_Number": 894, "Three_Letter_Country_Code": "ZMB", "Two_Letter_Country_Code": "ZM"},{"Continent_Code": "OC", "Continent_Name": "Oceania", "Country_Name": "Disputed Territory", "Country_Number": null, "Three_Letter_Country_Code": null, "Two_Letter_Country_Code": "XX"},{"Continent_Code": "AS", "Continent_Name": "Asia", "Country_Name": "Iraq-Saudi Arabia Neutral Zone", "Country_Number": null, "Three_Letter_Country_Code": null, "Two_Letter_Country_Code": "XE"},{"Continent_Code": "AS", "Continent_Name": "Asia", "Country_Name": "United Nations Neutral Zone", "Country_Number": null, "Three_Letter_Country_Code": null, "Two_Letter_Country_Code": "XD"},{"Continent_Code": "AS", "Continent_Name": "Asia", "Country_Name": "Spratly Islands", "Country_Number": null, "Three_Letter_Country_Code": null, "Two_Letter_Country_Code": "XS"}]
```

Comment:
Keys should be lowercase, reformat in a human readable way.
Also, is there really no python module that does this? I find this very very unlikely...